DQM/Integration/python/clients/onlinebeammonitor_dqm_sourceclient-live_cfg.py

```diff
@@ -144,7 +144,16 @@
 # Condition for lxplus: change and possibly customise the GT
 #from Configuration.AlCa.GlobalTag import GlobalTag as gtCustomise
 #process.GlobalTag = gtCustomise(process.GlobalTag, 'auto:run2_data', '')
-
+process.GlobalTag.toGet = cms.VPSet(
```

Comment:
I guess one line of comment explaining why this is needed would make it more robust against future accidental 'clean-up' of `toGet` statements.

Reply:
good point, I will add a brief description!